standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnErrorMsg.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.hive.metastore.txn;
+
+/**
+ * The following class represents all the error messages that are handled for aborts.
+ */
+public enum TxnErrorMsg {
+    // Txn Errors Codes: 50000 - 59999.
+    // Query runtime aborts - 50000-50999
+    NONE(50000, "None"),
+    ABORT_QUERY(50001, " Txn aborted by Abort Query Command"),
+    ABORT_CONCURRENT(50002, " Txn aborted due to concurrent committed transaction"),
+    ABORT_WRITE_CONFLICT(50003, " Txn aborted due to write conflicts"),
+    ABORT_TIMEOUT(50004, " Txn aborted due to heartbeat time-out"),
```

Comment:
some messages start with spaces and some don't, is it intentional?

Reply:
Removed spaces. Done.